README.md

```diff
@@ -0,0 +1,34 @@
+# apostrophe-link-widgets
+
+A link widget for [ApostropheCMS](http://apostrophecms.org/).
+
+`apostrophe-links-widgets` provides schema and a view for links.
+
+## Installation
+
+```bash
+npm install apostrophe-link-widgets --save
+```
+
+## Use
+
+You can use `apostrophe-links-widgets` as a singleton, area or as type `singleton` in your schema.
```

Comment:
"Use" should be "Usage." The sentence that follows does not quite make sense. It could be fixed but I don't think you need to teach the area/singleton distinction or the use of areas and singletons in schemas here. I would just say "You can use apostrophe-links widgets as you would any widget in apostrophe. here's an example..."

Reply:
Also it's apostrophe-link-widget, not apostrophe-links-widget. Please check pluralization consistency.